frontend/src/tests/HomePage.spec.js

```diff
@@ -0,0 +1,34 @@
+import Enzyme, { configure } from "enzyme";
```

Comment:
Nit: you don't need to import configure since you're referencing it by its fully qualified name below.